packages/demo/src/modules/clients/signals/addClient.js

```diff
@@ -6,6 +6,7 @@ export default [
   set(state`clients.$draft`, {}),
   set(state`clients.$draft.name`, state`clients.$filter`),
   set(state`clients.$draft.ref`, makeRef),
```

Comment:
WIth latest cerebral makeRef should be (just tested, it fixes an error on new client):

```js
import makeRef from '../../../helpers/ref'

export default function () {
  return {
    value: makeRef()
  }
}
```